app/Website/src/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Website\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Website\Http\Requests\ContactFormRequest;
+use Knuckles\Scribe\Attributes\BodyParam;
+use Knuckles\Scribe\Attributes\Endpoint;
+use App\Website\Mail\ContactMail;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Mail;
+
+class ContactController extends Controller
```

Comment:
As this controller only have one public action, and I don't think it will have anything else, you can make in with `public function __invoke()` and in the routes file you can make it like `Route::post('/contact', ContactController::class)`. Also maybe rename it to ContactUsController or ContactFormController so it is more clear.